python_modules/libraries/dagster-dbt/dagster_dbt/asset_decorator.py

```diff
@@ -299,6 +303,12 @@ def partitionshop_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource
         **(op_tags if op_tags else {}),
     }
 
+    if partitions_def and not backfill_policy:
+        if isinstance(partitions_def, TimeWindowPartitionsDefinition):
+            backfill_policy = BackfillPolicy.single_run()
+        elif isinstance(partitions_def, StaticPartitionsDefinition):
```

Comment:
Thoughts on turning this `elif` into an `else`?

Reply:
I think if `backfill_policy = None`, the default is multi-run once it resolves at the framework level? Wondering if we should just respect that default.

Can make it explicit if you think that's better.

Reply:
That also sounds reasonable to me. Just don't think we should treat `StaticPartitionsDefinition` specially relative to other non-time window partitions definitions.

Reply:
Got it. I removed that